documentation/GP_Fitness.java

```diff
@@ -0,0 +1,46 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class GP_Fitness {
+
+    private AST AST_tree;
+//    private
+
+    public GP_Fitness() {
+//        AST_tree = new AST<...>();
+    }
+//Patches here is a class that contains List (or ArrayList) of Insertion, Deletion, Replacement
+    public ArrayList<Patches> Loop_population(ArrayList<Patches> population){
```

Comment:
I assume you are trying to create an Individual obj here, right? Do not use ArrayList of Patches, there is an Individual class I created at the very beginning